app/scripts/services/membership/membership.js

```diff
@@ -114,18 +114,9 @@ angular
       return _.sortBy(roles, 'metadata.name');
     };
 
-    // TODO: follow-on PR, there will be an annotation for this
     var filterRoles = function(roles) {
       return _.filter(roles, function(item) {
-        // image-puller & image-pusher ok, other system: prob no
-        return (_.isEqual(item.metadata.name, 'system:image-puller')   ||
-                _.isEqual(item.metadata.name, 'system:image-pusher')   ||
-                _.isEqual(item.metadata.name, 'system:image-builder')  ||
-                _.isEqual(item.metadata.name, 'system:deployer')     ) ||
-                ! _.startsWith(item.metadata.name, 'cluster-') &&
-                ! _.startsWith(item.metadata.name, 'system:') &&
-                ! _.startsWith(item.metadata.name, 'registry-') &&
-                ! _.startsWith(item.metadata.name, 'self-');
+        return !annotation(item, 'systemOnly');
```

Comment:
you need to actually check the content of the annotation and compare it to the string 'true' in theory someone could set the annotation and set it to false